DS13/code/modules/overmap/transporter.dm

```diff
@@ -128,6 +160,19 @@
 			var/turf/open/Tu = get_turf(pick(orange(1, get_turf(eyeobj))))
 			T.send(Tu)
 		playsound(loc, 'DS13/sound/effects/transporter/send.ogg', 100, 4)
+	if(locked_on.len)
+		for(var/mob/living/H in locked_on)
+			var/area/AR = get_area(H)
+			if(!AR.linked_overmap)
+				continue
+			if(AR.linked_overmap.shields.check_vulnerability() || AR.linked_overmap.main_overmap) //If they don't have shields, or you want to site to site transport.
```

Comment:
What if `!AR.linked_overmap.shields` ?

Reply:
True, but that's an edge case

Reply:
Ideally you want your code to be robust enough to handle edge cases. Robust is an actual term from the industry, not a SS13 meme in this context.